class.jetpack-client.php

```diff
@@ -134,6 +134,10 @@ public static function remote_request( $args, $body = null ) {
 	 * @return array|WP_Error WP HTTP response on success
 	 */
 	public static function _wp_remote_request( $url, $args, $set_fallback = false ) {
+		if ( apply_filters( 'jetpack_disable_no_verify_ssl_certs', false ) ) {
```

Comment:
Could you add [some documentation](https://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/#4-hooks-actions-and-filters) before the filter? See [these examples](https://github.com/Automattic/jetpack/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Adocs+).